Tools/serial/generate_config.py

```diff
@@ -136,9 +136,9 @@
         "default_baudrate": 1, # set default to an unusable value to detect that this serial port has not been configured
         },
 
-    # Pixhawk Payload Bus
-    "PPB": {
-        "label": "Pixhawk Payload Bus",
+    # EXT2
```

Comment:
Does EXT1 exist? If so, can you add it as well?

Reply:
You would look on a schematic.

Yes EXT1 does exist but is only I2C3 (no serial).